core/chains/evm/log/helpers_test.go

```diff
@@ -104,15 +105,19 @@ func (c broadcasterHelperCfg) newWithEthClient(t *testing.T, ethClient evmclient
 	})
 	config := evmtest.NewChainScopedConfig(t, globalConfig)
 	lggr := logger.TestLogger(t)
+	mailMon := utils.NewMailboxMonitor(t.Name())
+	require.NoError(t, mailMon.Start(testutils.Context(t)))
+	t.Cleanup(func() { assert.NoError(t, mailMon.Close()) })
```

Comment:
Worth creating a `NullMailboxMonitor` that does no-ops?

Reply:
I kind of like exercising that logic and seeing the real overheads during the tests, but we could simplify this pattern with a general helper pretty easily, even a wrapper to inline with generics:
```go
package servicetest
func Start[S services.Service](*testing.T, s S) S {
	require.NoError(t, s.Start(testutils.Context(t)))
	t.Cleanup(func() { assert.NoError(t, s.Close()) })
	return s
}

mailMon := servicetest.Start(t, utils.NewMailboxMonitor(t.Name()))
```

Reply:
Updated with this.